source/includes/fact-save-pipeline.rst

```diff
@@ -1,24 +1,21 @@
-You can save a pipeline so that you can find it later. If you
-load a saved pipeline, you can change it without
-changing the original saved copy. You can also create a :ref:`view
-<view-details>` from your pipeline results.
-
-To save your pipeline:
-
 .. procedure::
    :style: normal
 
-   .. step:: Click the save dropdown button
+   .. step:: Click the save dropdown button.
 
-      Click the :guilabel:`Save` dropdown button at the top of the pipeline
-      builder. Click on ``Save pipeline as...``
+      In the aggregation pipeline pane, click the :guilabel:`Save` drop-down 
+      menu and select ``Save pipeline as...``.
 
       .. figure:: /images/compass/query-save-pipeline-as.png
          :alt: Save pipeline as
 
-   .. step:: Enter a name for your pipeline
+   .. step:: Enter a name for your pipeline.
 
       .. figure:: /images/compass/query-save-pipeline-as-name.png
          :alt: Name your pipeline
 
-   .. step:: Click :guilabel:`Save`
+   .. step:: Save the pipeline.
+
+      Click the :guilabel:`Save` button to save your pipeline. Your  pipeline 
+      will be saved under the :guilabel:`folder`` icon at the top left of the 
```

Comment:
[typo] There's an extra ` after your guilabel... but is this a guilabel? I would just leave this as a plain text icon description since "folder" isn't text that's in the UI itself.

Reply:
Noted and updated!